redcap_bridge/cli.py

```diff
@@ -28,18 +32,29 @@ def main(command_line=None):
                           help="Format to store the data (json/csv)")
     download.add_argument("-c", "--compressed", action='store_true',
                           help="Compress the output file (use labels and merge checkbox columns)")
+    download.add_argument("-s", "--server", type=str, nargs=1, metavar='server',
+                          choices=['redcap', 'elabftw'], help="The two server choices are redcap or elabftw",
+                          default='redcap')
+    download.add_argument("experiment_id", nargs='?', metavar='experiment_id', type=int,
```

Comment:
I think this needs to be a keyword based argument, so we can make it required when `--server elabftw`, no?

Reply:
I got the same error, I need to go in details with the required argument because I don't know why it doesn't work in our particular case